frontend/src/components/css/Login.scss

```
@@ -0,0 +1,82 @@
+
+.login-content {
+    height: 90%;
+    width: 90%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.login-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 60%;
+    width: 40%;
+    background-color: white;
+    padding: 2.5%;
+}
+
+.login-text {
+    display: flex;
+    color: #644A0A;
+    font-weight: bold;
+    font-size: 150%;
+    height: 6%;
+}
+
+.login-input-container {
+    display: flex;
+    flex-direction: column;
+    border-bottom: 3px solid #828282;
+}
+
+.login-input-container__clicked {
+    display: flex;
+    flex-direction: column;
+    border-bottom: 3px solid #E4BB4F;
+}
+
+.login-input {
+    font-size: 100%;
+    margin-left: -1.5%;
+}
+
+.login-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+}
+
+.login-register-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+
+.login-register-button {
+    background: none;
+    border: none;
+    color: #E4BB4F;
+    text-decoration: underline;
+    margin-left: 5px;
+    transition-duration: 0.4s;
+}
+
+.login-register-button:hover {
+    background: none;
+    border: none;
+    color: #ffe5a3;
+    text-decoration: underline;
+    margin-left: 5px;
+}
+
+.login-register-button:active {
+    background: none;
+    border: none;
+    color: #E4BB4F;
+    text-decoration: underline;
+    margin-left: 5px;
+}
```

this css is kinda redundant since you can nest
```suggestion
.login-register-button {
    background: none;
    border: none;
    color: #E4BB4F;
    text-decoration: underline;
    margin-left: 5px;
    transition-duration: 0.4s;
    
    :hover {
          color: #ffe5a3;
    }
    
    :active {
          color: #E4BB4F;
     }
}
```

---

also please add color constants to `frontend/src/components/css/_app.scss` and import to use variable